src/pages/swap/helpers/swapConfirm/index.ts

```diff
@@ -0,0 +1,92 @@
+import { StateUpdater } from 'preact/hooks';
+import { ApiPromise } from '@polkadot/api';
+import Big from 'big.js';
+
+import {
+  getInputTokenDetailsOrDefault,
+  InputTokenType,
+  OutputTokenType,
+  OUTPUT_TOKEN_CONFIG,
+} from '../../../../constants/tokenConfig';
+
+import { ExecutionInput } from '../../../../hooks/offramp/useMainProcess';
+import { TokenOutData } from '../../../../hooks/nabla/useTokenAmountOut';
+import { Networks } from '../../../../contexts/network';
+
+import { calculateSwapAmountsWithMargin } from './calculateSwapAmountsWithMargin';
+import { performSwapInitialChecks } from './performSwapInitialChecks';
+import { validateSwapInputs } from './validateSwapInputs';
+
+interface SwapConfirmParams {
```

Comment:
Let's at least order the params alphabetically or logically as they are a wild mix at the moment.